aws_db_instance.tf

```diff
@@ -3,47 +3,53 @@
 #}
 
 resource "aws_db_instance" "secureInstance" {
-  allocated_storage    = 20
-  storage_type         = "gp2"
-  engine               = "mysql"
-  engine_version       = "5.7"
-  instance_class       = "db.t2.micro"
-  name                 = "mydb"
-  backup_retention_period = 90
+  allocated_storage                   = 20
+  storage_type                        = "gp2"
+  engine                              = "mysql"
+  engine_version                      = "5.7"
+  instance_class                      = "db.m5.xlarge"
+  name                                = "mydb"
+  backup_retention_period             = 90
   iam_database_authentication_enabled = true
-  auto_minor_version_upgrade = true
+  auto_minor_version_upgrade          = true
+  username                            = "<some_random_username>"
 }
 
 resource "aws_db_instance" "rdsBackupDisbaled" {
-  allocated_storage    = 20
-  storage_type         = "gp2"
-  engine               = "mysql"
-  engine_version       = "5.7"
-  instance_class       = "db.t2.micro"
-  name                 = "mydb"
```

Comment:
**Accurics remediation**
 
**[HIGH]: RDS Instance publicly_accessible flag is true**
 
**Vulnerability**
RDS Instance publicly_accessible flag is true
 
**Remediation**
Ensure RDS instances are not public facing to minimise security risks

source: accurics-